azure/packages/azure-client/.eslintrc.js

```diff
@@ -9,60 +9,4 @@ module.exports = {
     parserOptions: {
         project: ["./tsconfig.json", "./src/test/tsconfig.json"],
     },
-    rules: {
```

Comment:
All of these removed rules and overrides are now reflected in the base config, and are therefore no longer needed here.